content/spin/kv-store.md

```diff
@@ -0,0 +1,355 @@
+title = "Using Key Value Storage with Spin Applications"
+template = "spin_main"
+date = "2023-02-21T00:00:00Z"
+enable_shortcodes = true
+[extra]
+
+---
+- [Using Key Value Storage With Spin Applications](#using-key-value-storage-with-spin-applications)
+    - [Prerequisites](#prerequisites)
+    - [Use Spin SDK To Open And Persist Data In A Default Key/Value Store](#use-spin-sdk-to-open-and-persist-data-in-a-default-keyvalue-store)
+    - [Configure `spin.toml` To Use A Default Key/Value Store](#configure-spintoml-to-use-a-default-keyvalue-store)
+    - [Build And Deploy Your Spin App](#build-and-deploy-your-spin-app)
+    - [Store and Retrieve Data From Your Default Key/Value Store](#store-and-retrieve-data-from-your-default-keyvalue-store)
+    - [Conclusion](#conclusion)
+    - [Next Steps](#next-steps)
+
+## Using Key Value Storage with Spin Applications
+
+ Spin application are best suited for event-driven, stateless workloads that have low-latency requirements. If your Spin applications needs to persist state, the data will need to be stored in an external tool or service. In [Spin v0.9.0](https://www.fermyon.com/blog/spin-v09), we've added support for key/value storage. This is a storage option for applications that need to data in the form of key/value pairs and are satistifed by a BASE consistency model. Workload examples include general value caching, session caching, counters, serialized application state.
```

Comment:
Spin _applications_.

Reply:
I think the consistency model referenced needs at least a link.

I'd be inclined to detail the consistency model section and move it to a separate section at the end.

Reply:
Agreed, we should go into more detail. I'm thinking it may make sense to have a separate conceptual article that walks the user through the nuts and bolts of the KV implementation, what its intended use case is, consistency model, any limitations to be aware of, etc.

Reply:
Thanks, Radu, this has been reworded in the new push and Kenzie, and I have a more conceptual article in the works https://github.com/fermyon/developer/issues/364